src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/process/DefaultProcessRunner.java

```diff
@@ -99,14 +80,6 @@ public void run(String[] command) throws ProcessRunnerException {
       // Configure process builder.
       final ProcessBuilder processBuilder = new ProcessBuilder();
 
-      // If there are no listeners, we might still want to redirect stdout and stderr to the parent
-      // process, or not.
-      if (stdOutLineListeners.isEmpty() && inheritProcessOutput) {
```

Comment:
I just realized that if someone was using a start listener to get directly at the process's output, this would break them.